model/clusters_mgmt/v1/autoscaler_request_type.model

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright (c) 2023 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Contains the necessary attributes to allow creation of cluster autoscalers
+struct ClusterAutoscalerRequest {
```

Comment:
Why `request`? we usually just write the name of the type.

Reply:
Everywhere I'm seeing we're doing this in the POST-requests definition:
```
method Post {
    in Request <object>Request
```

there are not a lot of cases where we write something else (I could only find ``SubscriptionRegistration`` and ``LabelRequestPayload``)

Reply:
I can't comment about why AMS is using "X-Request" pattern. But for CS, we don't use this convention. The only exception that I found is STSCredentialsRequest, but that's because the endpoint itself contains "request" in the URL, so it's aligns with our conventions.

Reply:
ack. I changed it to ``ClusterAutoscaler``